cloud_cleaner.sh

```diff
@@ -63,7 +63,7 @@ for i in $(seq 0 $(("${#RESOURCE_TYPES[@]}" - 1))); do
 done
 
 # Explicitly check the other storage accounts (mostly the api test one)
-STORAGE_ACCOUNT_LIST=$(az resource list -g "$AZURE_RESOURCE_GROUP" --resource-type Microsoft.Storage/storageAccounts)
+STORAGE_ACCOUNT_LIST=$(az resource list -g "$AZURE_RESOURCE_GROUP" --resource-type Microsoft.Storage/storageAccounts | jq -r "select(.[].tags.\"persist\" != \"true\")")
```

Comment:
Storage blobs don't have tags. Thay have metadata, but for some reason, the cli always says there's no metadata...

I also thought it would be more convenient. As someone might want to upload a lot of blobs, and adding the metadata to each of them is not great